src/scripts/README.md

```diff
@@ -74,6 +75,9 @@
 ### Client Isolation:
     create_ap --isolate-clients wlan0 eth0 MyAccessPoint MyPassPhrase
 
+### Custom DHCP scope
+    create_ap wlp8s0 MyAccessPoint MyPassPhrase --dhcp-range 150,155
```

Comment:
If you split the option as mentioned below, you need to update this example also ofcourse :wink: